core/src/main/java/com/linecorp/armeria/common/stream/TimeoutSubscriber.java

```diff
@@ -29,70 +29,88 @@
 import io.netty.util.concurrent.ScheduledFuture;
 
 public class TimeoutSubscriber<T> implements Subscriber<T> {
```

Comment:
Can we hide this class from the public API? i.e. `final class TimeoutSubscriber ...`